automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -776,3 +778,20 @@ def csv_reader(component, subcommand):
         entity_list.append(row)
     comp_dict[component] = entity_list
     return comp_dict
+
+
+def create_setup_dict(dict):
+    data.update(dict)
```

Comment:
Remove this as we are removing data variable. Not required as we are directly passing dictionary to json dump.